src/schema/objects/metadata.yaml

```diff
@@ -2554,6 +2696,18 @@ RecordingType:
     - continuous
     - epoched
     - discontinuous
+RecordTimeStart:
```

Comment:
Maybe reuse `TimeZero` (similar to the one used in PET) eventually with a new definition.

Reply:
Or maybe [StartTime](https://bids-specification--1128.org.readthedocs.build/en/1128/glossary.html#objects.metadata.StartTime) like for physio stuff

Reply:
I changed `RecordTimeStart` to `StartTime_eyetrack` (keeping the name tag StartTime as in Physio -> sense 2) and for clarity changed `RecordTimeEnd` to newly created `StopTime_eyetrack` (stop being a better antonym of Start than End).